electron-app/src/views/OtcTrades.vue

```diff
@@ -182,11 +182,6 @@ export default class OtcTrades extends Vue {
     this.$api
       .deleteExternalTrade(this.deleteId)
       .then(() => {
-        this.$store.commit('setMessage', {
```

Comment:
This is what you asked for in the chat I guess? We still get the visual confirmation, for this,right?

Reply:
Well the list updates, so this would be enough visual confirmation right? You delete an entry and it disappears.

Reply:
If the list updates and you are looking at it in the moment it updates I guess yes. The trade can't be in another page, since you are the one deleting/editing it so probably works